laser_line_extraction_lifecycle:
  ros__parameters:
    frequency: 80.0
    frame_id: "hokuyo_scan_link"
    scan_topic: "scan"
    publish_markers: true
    bearing_std_dev: 1e-5
    range_std_dev: 0.012
    least_sq_angle_thresh: 0.0001
    least_sq_radius_thresh: 0.0001
    max_line_gap: 0.1
    min_line_length: 0.2
    min_range: 0.1
    max_range: 250.0
    min_split_dist: 0.04
    outlier_dist: 0.06
    min_line_points: 1